webapi/src/main/resources/knora-ontologies/knora-base.ttl

```diff
@@ -2531,11 +2531,13 @@
 
 ### http://www.knora.org/ontology/knora-base#TextValueType
 
-:TextValueType a owl:Class ;
-    owl:oneOf ( :UnformattedText
-                :FormattedText
-                :CustomFormattedText
-                :UndefinedTextType ) .
+:TextValueType ovl:equivalentClass [ a owl:Class ;
```

Comment:
`owl` typeo

Otherwise LGTM, but as I said, I have never used it in the real world.

Reply:
thanks for spotting! 🙈